### ISSUE TITLE
Blobfuse drive not working properly

### ISSUE THREAD

**Issue by rajancsp** [Azure/azure-storage-fuse#197]
https://docs.microsoft.com/en-us/azure/storage/blobs/storage-how-to-mount-container-linux 

BlobFuse drive Mounting on client machine(linux 7.2), refer to following document and configured in Azure Virtual Machine and Client machine. 

Azure Virtual Machine : 
================
I don't see any issue on Azure VM like copy from local to blobfuse drive and vice versa. 

Client Linux Machine : 
=================

- So far, configured blobfuse and mounted the drive on client machine 

blobfuse dhlprodoutputfile --tmp-path=/mnt/resource/azdhlprodoutput --config-file=/mnt/resource/azdhlprodoutput.cfg -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120 --use-https=true

-when we try to copy from local machine to blobfuse taking time even 1mb file takes 5-6 min, after few min file not there in blobfuse drive and not showing up in storage explorer. 

please advice to fix this issue.

**Comment by seguler** [comment]
So you are saying it works fine on Azure VMs, but not on-prem machines ?

Can you enable logging as per [this document](https://github.com/Azure/azure-storage-fuse/wiki/5.-Logging) and share the logs with us ?

**Comment by rajancsp** [comment]
not all the time its working on/off not like works fine as Azure VMs. 

Already following the document as below, but not able to find the logs where it's generate. Please advice on this....

sudo blobfuse dhlprodinputfile --tmp-path=/blobfuse/azdhlprodinput --config-file=/blobfuse/azdhlprodinput.cfg -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120 --use-https=true --log-level=LOG_WARNING --log-level=/blobfuse/LOG_INFO --log-level=LOG_DEBUG

**Comment by seguler** [comment]
Logs will be in /var/log/syslog

Just grep ```blobfuse /var/log/syslog```

Did you say it works on Azure VMs, but fails occasionally in the client machine outside Azure ?

**Comment by rajancsp** [comment]
Yes it works well on Azure VMs...

I don't see any files creation like that..."grep blobfuse /var/log/syslog"

**Comment by seguler** [comment]
So two minor problems I see:

1. You just need to set log-level once. Use the following code to set it to DEBUG mode.

`sudo blobfuse dhlprodinputfile --tmp-path=/blobfuse/azdhlprodinput --config-file=/blobfuse/azdhlprodinput.cfg -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120 --use-https=true --log-level=LOG_DEBUG`

2. You are using sudo - meaning the root user will mount it. But you do not have -o allow_other flag which would be needed for non-root users to access the mount point. Try this:

`sudo blobfuse dhlprodinputfile --tmp-path=/blobfuse/azdhlprodinput --config-file=/blobfuse/azdhlprodinput.cfg -o allow_other -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120 --use-https=true --log-level=LOG_DEBUG`

If this doesn't work, try mounting with your user (without sudo)

**Comment by rajancsp** [comment]
Options 2: 
am mounting the drive as root user, but need to be access other user as well....

Let me try again, with both options and update you same... by default logs will generate this path /var/log/syslog

**Comment by seguler** [comment]
No response - closing
